cmd/clusterlink/cmd/create/create_peer.go

```diff
@@ -242,14 +242,14 @@ func (o *PeerOptions) Run() error {
 	return os.WriteFile(clOutPath, clConfig, 0o600)
 }
 
-// NewCmdCreatePeer returns a cobra.Command to run the 'create peer' subcommand.
+// NewCmdCreatePeer returns a cobra.Command to run the 'create peer-cert' subcommand.
 func NewCmdCreatePeer() *cobra.Command {
 	opts := &PeerOptions{}
 
 	cmd := &cobra.Command{
-		Use:   "peer",
-		Short: "Create a peer",
-		Long:  `Create a peer`,
+		Use:   "peer-cert",
+		Short: "Create peer certificates",
```

Comment:
nit: actually it creates a *single* certificate, not plural.
you could change to `Create peer certificate and private key" if you're so inclined, but not needed.